lecture5/.gitignore

```diff
@@ -1,6 +0,0 @@
-emptyfile
```

Comment:
@DeekshithPatil I need to keep the existing directories, since these are referenced in lecture content.  Need to revert the portions of https://github.com/cu-ecen-aeld/aesd-lectures/pull/2/commits/fe0a41d9c84391faa7fbde397377fcfebdbad30e which deleted these.